specification/data-semantic-conventions.md

```diff
@@ -14,49 +14,125 @@ This way, the operator will not need to learn specifics of a language and
 telemetry collected from multi-language micro-service can still be easily
 correlated and cross-analyzed.
 
-## HTTP client
-
-This span type represents an outbound HTTP request.
-
-For a HTTP client span, `SpanKind` MUST be `Client`.
-
-Given an [RFC 3986](https://www.ietf.org/rfc/rfc3986.txt) compliant URI of the form
+**Table of contents**
+
+- [HTTP](#http)
+  * [HTTP client](#http-client)
+  * [HTTP server](#http-server)
+- [Databases client calls](#databases-client-calls)
+- [Remote procedure calls](#remote-procedure-calls)
+  * [Attributes](#attributes)
+  * [gRPC](#grpc)
+    + [Status](#status)
+    + [Events](#events)
+- [Messaging systems](#messaging-systems)
+  * [Definitions](#definitions)
+  * [Conventions](#conventions)
+  * [Messaging attributes](#messaging-attributes)
+- [General attributes](#general-attributes)
+  * [General miscellaneous attributes](#general-miscellaneous-attributes)
+  * [General source code attributes](#general-source-code-attributes)
+  * [General network connection attributes](#general-network-connection-attributes)
+
+## HTTP
+
+This span types represents HTTP requests. They can be used for http and https
+schemes and various HTTP versions like 1.1, 2 and SPDY.
+
+Given an [RFC 3986](https://tools.ietf.org/html/rfc3986) compliant URI of the form
 `scheme:[//authority]path[?query][#fragment]`, the span name of the span SHOULD
 be set to to the URI path value.
 
-If a framework can identify a value that represents the identity of the request
-and has a lower cardinality than the URI path, this value MUST be used for the span name instead.
+If a value that represents the identity of the request
```

Comment:
This sentence is confusing. I'd prefer something along the lines of:

"Otherwise, if a value can be obtained that represents the identity of the request and has a lower cardinality than the URI path, this value MUST be used for the span name instead."

Reply:
It's not really "otherwise" though, as a URI path can still be used.

Reply:
Reworded.